### ISSUE TITLE
[1.20.1] Instant client disconnect when loading into a server.

### ISSUE THREAD

**Issue by Gbergz** [canitzp/FeederHelmet#34]
## Title^

**MC:** 1.20.1
**Forge:** 47.0.35
**Auto Feeder Helmet:** 47.0.0 | https://www.curseforge.com/minecraft/mc-mods/auto-feeder-helmet/files/4622479

part of logs: https://gist.githubusercontent.com/Gbergz/97484f21a8c8281d4b3693089e667ab0/raw/f5a515a72a0950d05dbacedcabf72da95218f501/gistfile1.txt

![screenshot](https://github.com/canitzp/FeederHelmet/assets/26634554/3cd5a966-2a8b-4ada-b121-f00aa2461ae4)

**-** Gbergz

**Comment by canitzp** [comment]
Just so you know, I have identified the problem, pretty obvious, but well my testing lacks a lot. This does affect Feeder Helmet and Solar Helmet since 1.20, so I'll probably release a 46.x.0, with support for both 1.20 and 1.20.1.

**Comment by canitzp** [comment]
So this issue is now fixed as well. I decided against compatibility with 1.20, so the new version 47.0.1 is no available on modrinth and soon on curseforge.
Many thanks again for telling me these issues, you're not the first one to download the affected versions, but the first to report issues. If you have any other issues create a new issue. If you have a mod idea/feature suggestion, tell me in a seperate issue or write me on discord.
